bindings/java/src/org/sleuthkit/datamodel/TagName.java

```diff
@@ -33,39 +33,45 @@
 
 	public enum HTML_COLOR {
 
-		NONE("None"), //NON-NLS
-		WHITE("White"), //NON-NLS
-		SILVER("Silver"), //NON-NLS
-		GRAY("Gray"), //NON-NLS
-		BLACK("Black"), //NON-NLS
-		RED("Red"), //NON-NLS
-		MAROON("Maron"), //NON-NLS
-		YELLOW("Yellow"), //NON-NLS
-		OLIVE("Olive"), //NON-NLS
-		LIME("Lime"), //NON-NLS
-		GREEN("Green"), //NON-NLS
-		AQUA("Aqua"), //NON-NLS
-		TEAL("Teal"), //NON-NLS
-		BLUE("Blue"), //NON-NLS
-		NAVY("Navy"), //NON-NLS
-		FUCHSIA("Fuchsia"), //NON-NLS
-		PURPLE("Purple"); //NON-NLS
+		NONE	("None",	""), //NON-NLS
+		WHITE	("White",	"#FFFFFF"), //NON-NLS
+		SILVER	("Silver",	"#C0C0C0"), //NON-NLS
+		GRAY	("Gray",	"#808080"), //NON-NLS
+		BLACK	("Black",	"#000000"), //NON-NLS
+		RED		("Red",		"#FF0000"), //NON-NLS
+		MAROON	("Maron",	"#800000"), //NON-NLS
+		YELLOW	("Yellow",	"#FFFF00"), //NON-NLS
+		OLIVE	("Olive",	"#808000"), //NON-NLS
+		LIME	("Lime",	"#00FF00"), //NON-NLS
+		GREEN	("Green",	"#008000"), //NON-NLS
+		AQUA	("Aqua",	"#00FFFF"), //NON-NLS
+		TEAL	("Teal",	"#008080"), //NON-NLS
+		BLUE	("Blue",	"#0000FF"), //NON-NLS
+		NAVY	("Navy",	"#000080"), //NON-NLS
+		FUCHSIA	("Fuchsia", "#FF00FF"), //NON-NLS
+		PURPLE	("Purple",	"#800080"); //NON-NLS
```

Comment:
What do the '#' chars signify in these strings?

Reply:
The # is part of the hex value.  I think it has something to do with the HTML specification?  I quick google, didn't tell me why the # is there other than its expected by the <style> tag.

Reply:
It sounds like the '#' character is HTML-markup specific, then. I imagine you are trimming it off in IG code I haven't seen yet. I think we could probably drop it from the enum, but I don't think it much matters.

Reply:
Nope.  I don't trim it.  The color methods in both java se and javafx take it with the #.  In fact if we didn't have it on there I think I would have to add it, maybe.